dive_core/generatePm4Info_adreno.py

```diff
@@ -11,36 +11,36 @@
 
 # ---------------------------------------------------------------------------------------
 def outputHeader(pm4_info_file):
-  pm4_info_file.writelines(
-    " /*\n"
-    " Copyright 2020 Google LLC\n"
-    "\n"
-    " Licensed under the Apache License, Version 2.0 (the \"License\";\n"
-    " you may not use this file except in compliance with the License.\n"
-    " You may obtain a copy of the License at\n"
-    "\n"
-    " http://www.apache.org/licenses/LICENSE-2.0\n"
-    "\n"
-    " Unless required by applicable law or agreed to in writing, software\n"
-    " distributed under the License is distributed on an \"AS IS\" BASIS,\n"
-    " WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n"
-    " See the License for the specific language governing permissions and\n"
-    " limitations under the License.\n"
-    "*/\n"
-    "\n"
-    "///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////\n"
-    "//\n"
-    "// WARNING!  WARNING!  WARNING!  WARNING!  WARNING!  WARNING!  WARNING!  WARNING!  WARNING!  WARNING!  WARNING!\n"
-    "//\n"
-    "// This code has been generated automatically by generatePm4Strings_adreno.py. Do not hand-modify this code.\n"
-    "//\n"
-    "///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////\n"
-  )
+  pm4_info_file.writelines('''
+     /*
+     Copyright 2020 Google LLC
+    
+     Licensed under the Apache License, Version 2.0 (the "License";
```

Comment:
Add missing closed bracket to align with the text in the LICENSE
```suggestion
     Licensed under the Apache License, Version 2.0 (the "License");
```